server/api/contest_handler.py

```diff
@@ -0,0 +1,56 @@
+import json
+from flask import jsonify, Blueprint, request
+from api.models import contest_table, db
+from datetime import date
+contest_handler = Blueprint('contest_new_handler', __name__)
+
+
+@contest_handler.route('/create_new_contest', methods=['POST'])
+def create_contest():
+    if request.method == 'POST':
+
+        title = request.form['title']
```

Comment:
Should we use JSON or Form data when sending post requests?

This is mostly concern about the consistency across the api.

Reply:
Actually now that I think about it more, because it's likely that we will have to customize a lot of these fields when sending it through React, it's most likely it's going to be JSON. you don't have to change it now, but when we integrate this it's probably going to change